index.html

```diff
@@ -278,7 +287,7 @@ <h3 class="skill-title">Frameworks</h3>
           </ul>
         </li>
         <li>
-          <ul class="skill-wrapper">
+          <ul class="skill-wrapper last-skill-wrapper">
             <li class="skill-title-wrapper">
               <svg
                 class="skill-title-svg"
```

Comment:
- [ ] I think it would be better if you improve the skills, languages, Frameworks list design by making them be aligned at `left` like on the template design. This will make your design be close to your chosen template and for project requirements.